sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-26.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Maintainers Minutes: August 2024'
+subtitle: 'Keeping you informed of the latest maintainers discussions'
+pubDate: 2024-08-14T12:00:00+02:00
+headerImage: '/assets/images/blog/maintainers-minutes-2024-07-26/maintainers_duck.png'
+headerImageAlt: "Cartoon yellow rubber duck with nf-core logo badge on it's body with the nf-core logo."
+embedHeaderImage: true
+authors:
+    - 'jfy133'
+    - 'FriederikeHanssen'
+label:
+    - 'maintainers'
+---
+
+The 'Maintainers Minutes' aims to further give insight into the workings of the [nf-core maintainers team](/governance#maintainers)
+by providing brief summaries of the monthly team meetings.
+
+## Review of action points from last time
+
+We got about half way through our action points from last time.
+
+Slack channels were created for the new working groups: - #wg-meta-pipelines - #wg-modules-update - #wg-test-data-task-force
+
+Please join if you want to be involved in one of those.
```

Comment:
```suggestion
```

I think we already covered this in the previous blog post, and reviewing last meetings points isn't so interesting to readers (I think anyway)